src/shared/constants.ts

```
@@ -83,6 +83,33 @@
 
 export type EnvKey = keyof typeof ENV_KEYS;
 
+const llms = [
```

ESLint

[error] 'llms' is assigned a value but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)